{"name": "test_login_fail[case1]", "status": "passed", "parameters": [{"name": "case", "value": "{'username': 'taozi', 'passwd': '123456', 'check': 'Invalid username or password. Signon failed.'}"}], "start": 1665473634458, "stop": 1665473635637, "uuid": "ccb65772-4a6e-4127-9923-560715803214", "historyId": "ce469f5486b41c2a0f55aee02de55945", "testCaseId": "8c4a898127c5b4ed67fea7eda153ddc9", "fullName": "Test_Cases.test_login.Test_login#test_login_fail", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init')"}, {"name": "parentSuite", "value": "Test_Cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "DESKTOP-41KMUJO"}, {"name": "thread", "value": "14644-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_Cases.test_login"}]}